program/c/src/oracle/upd_aggregate.h

```diff
@@ -174,8 +176,7 @@ static inline bool upd_aggregate( pc_price_t *ptr, uint64_t slot, int64_t timest
       if ( iptr->agg_.status_ == PC_STATUS_TRADING &&
            // No overflow for INT64_MIN+conf or INT64_MAX-conf as 0 < conf < INT64_MAX
            // These checks ensure that price - conf and price + conf do not overflow.
-           (int64_t)0 < conf && (INT64_MIN + conf) <= price && price <= (INT64_MAX-conf) &&
-           slot_diff >= 0 && slot_diff <= PC_MAX_SEND_LATENCY ) {
+           (int64_t)0 < conf && (INT64_MIN + conf) <= price && price <= (INT64_MAX-conf) && slot_diff <= PC_MAX_SEND_LATENCY ) {
```

Comment:
removed `slot_diff >= 0` here because there is a check above already that makes sure slot_diff is >= 0
```
  if ( slot < ptr->agg_.pub_slot_ ) {
    return false;
  }
  ```